codalab/rest/users.py

```diff
@@ -143,6 +143,44 @@ def fetch_users():
     users = local.model.get_users(usernames=(usernames or None))
     return allowed_user_schema()(many=True).dump(users).data
 
+@get('/userstats')
```

Comment:
Oh, we already have a `/users` endpoint...where is `/users` used?  We should unify the two and call it `/users` since it seems like your new version is strictly more powerful?

Reply:
We rely on `/users` for fetching specific users by username and email directly, whereas `/userstats` fetches users by keywords in the form of username=codalab. Do we really want to combine the two?